index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
```

Comment:
For the <html> element, the lang attribute should indicate the appropriate document language.